pkgs/top-level/all-packages.nix

```diff
@@ -22021,7 +22021,9 @@ with pkgs;
 
   mtpfs = callPackage ../tools/filesystems/mtpfs { };
 
-  mtxclient = callPackage ../development/libraries/mtxclient { };
+  mtxclient = callPackage ../development/libraries/mtxclient {
+    stdenv = if stdenv.isLinux && stdenv.isAarch64 && stdenv.cc.isGNU then gcc11Stdenv else stdenv;
```

Comment:
Links to https://github.com/NixOS/nixpkgs/issues/201254 should also be added here, so that we can easily revert these hacks later.

Reply:
Done.